yarpl/include/yarpl/flowable/FlowableOperator.h

```diff
@@ -700,6 +757,13 @@ class FlatMapOperator : public FlowableOperator<T, R, FlatMapOperator<T, R>> {
       else {
         requested_ += n;
       }
+
+      if (n > 0) {
+        // TODO: make max parallelism configurable a-la RxJava 2.x's
+        // FlowableFlatMapOperator
+        SuperSubscription::request(std::numeric_limits<int64_t>::max());
```

Comment:
For the short term I think this should be int32_t::max(), cause we have a lot of bad assertions in rsocket that assume that.